filters/shedder/admission.go

```diff
@@ -315,8 +315,8 @@ func (spec *AdmissionControlSpec) CreateFilter(args []interface{}) (filters.Filt
 		averageRpsFactor: averageRpsFactor,
 		totals:           make([]int64, windowSize),
 		success:          make([]int64, windowSize),
-		counter:          atomic.NewInt64(0),
-		successCounter:   atomic.NewInt64(0),
+		counter:          &atomic.Int64{},
```

Comment:
It could also be
```go
counter: new(atomic.Int64),
```

Reply:
I've added `new()` in the new patchset. Out of curiosity, is there a reason for refactoring this to `new` based initialization? 🤔

Reply:
No, `&atomic.Int64{}` is equivalent to `new(atomic.Int64)`

Reply:
Although they are equivalent you should stick to one of your taste.

Reply:
Got it, thanks 🙂